src/HotChocolate/Data/src/Data/Projections/ISelectionVisitorContext.cs

```diff
@@ -12,6 +12,8 @@ public interface ISelectionVisitorContext
 
         Stack<SelectionSetNode?> SelectionSetNodes { get; }
 
+        Stack<INamedType?> ResolvedType { get; }
```

Comment:
@michaelstaib  Should this wait till 12?

Reply:
would be better

Reply:
@michaelstaib Could this be merged and made part of preview release please?